Jenkinsfile

```diff
@@ -59,20 +59,16 @@ pipeline {
 	       		}
 	     	}
 	    }
-	    
-    	stage('Build and Package') {
-      		steps {
-      			script {
-					withCredentials([string(credentialsId: 'sonar-token-capella', variable: 'SONARCLOUD_TOKEN')]) {
-						withEnv(['MAVEN_OPTS=-Xms3000m -Xmx3000m']) {
-							def sign = github.isPullRequest() ? '' : '-Psign'
-							sh "mvn verify -Pfull ${sign}"
-						}
-					}
-      			}
-	     	}
-	    }
-	    
+
+		stage('Build and Package') {
+			steps {

```

Comment:
i assume sonar is being entirely taken care of in its separate stage so no need for credentials?